### ISSUE TITLE
[Lightsail Bitnami Wordpress] "<VirtualHost> was not closed" error when running bncert-tool

### ISSUE THREAD

**Issue by Rtax** [bitnami/vms#1062]
I am running a multisite Wordpress installation. I am generating a LetsEncrypt certificate for all the domains. If I add a site, I will run the bncert-tool again and put in all the domains. It runs without problems, but then fails on Apachectl configtest, and all changes are reverted.

The error in the log is:
httpd: Syntax error on line 518 of /opt/bitnami/apache/conf/httpd.conf: Syntax error on line 1 of /opt/bitnami/apache/conf/vhosts/wordpress-https-vhost.conf: /opt/bitnami/apache/conf/vhosts/wordpress-https-vhost.conf:1: <VirtualHost> was not closed.

I have checked all the files in vhosts and all <VirtualHost> tags are closed correctly.

### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

1bd3c21b-d2d5-8f1b-8de2-de3dd462aa6d

### bndiagnostic output

 ? Resources: Found possible issues
    ✓ Connectivity: No issues found
    ✓ Mariadb: No issues found
    ✓ Processes: No issues found
    ✓ Wordpress: No issues found
    ? Apache: Found possible issues
    ✓ Php: No issues found
 

 
[Resources]

Your instance has little available RAM memory.
```
 total used free shared buff/cache available Mem: 975 501 70 86 403 221 Swap: 
634 47 587
```
You could try to increase your instance's memory. Please check your cloud 
provider's documentation for more information.
Press [Enter] to continue:

[Apache]

Found recent error or warning messages in the Apache error log.
```
[Tue Jul 25 14:37:56.641564 2023] [proxy_fcgi:error] [pid 1228063:tid 
140360764172032] [client **ip_address**:35960] AH01071: Got error 'PHP message: 
PHP Warning: Constant WP_MEMORY_LIMIT already defined in 
/bitnami/wordpress/wp-config.php on line 229'
 [Tue Jul 25 14:38:49.955486 2023] [proxy_fcgi:error] [pid 1228325:tid 
140361141843712] [client **ip_address**:6566] AH01071: Got error 'PHP message: 
PHP Warning: Constant WP_MEMORY_LIMIT already defined in 
/bitnami/wordpress/wp-config.php on line 229'
 [Tue Jul 25 14:38:54.369792 2023] [proxy_fcgi:error] [pid 1228325:tid 
140360822920960] [client **ip_address**:42444] AH01071: Got error 'PHP message: 
PHP Warning: Constant WP_MEMORY_LIMIT already defined in 
/bitnami/wordpress/wp-config.php on line 229'

### bndiagnostic was not useful. Could you please tell us why?

The error is with bncert tool

### Describe your issue as much as you can

**Comment by gongomgra** [comment]
Hi @Rtax,

Thanks for using Bitnami. Can you tell us if you are following any guide for adding the new site? If so, can you share the link with us? Can you also try to generate the SSL certificate manually following the next guide?

https://docs.bitnami.com/aws/how-to/generate-install-lets-encrypt-ssl/#alternative-approach

**Comment by Rtax** [comment]
Hi. I used this guide: https://docs.bitnami.com/aws/how-to/generate-install-lets-encrypt-ssl/ which worked fine. The site is working and the certificates do update. Even though the final part of the script fails.

Everything was very standard, the only odd thing I can think of is that I did change the domain name of the main multisite at a point. i.e Main site was abc.com, and is now xyz.com. I also put all the subsite domains in the certificate, so there are 11 domains in the certificate.
